### PR TITLE
AMD ROCm rocm-gdb update with its dependency package - babeltrace

### PR DESCRIPTION
**Pull request by arjun-raj-kuppala**

### CODE REVIEW

**Comment by arjun-raj-kuppala** [comment]
@adamjstewart , Can you review this ?

File: var/spack/repos/builtin/packages/rocm-gdb/package.py [hunk 1]
```diff
@@ -28,6 +28,8 @@ class RocmGdb(AutotoolsPackage):
     depends_on('expat', type='build')
     depends_on('python', type='build')
     depends_on('zlib', type='link')
+    depends_on('babeltrace@1.2.4', type='link')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/rocm-gdb/package.py:31]: Is this the only version of babeltrace that works? If not, I would remove the version specification.

Comment (arjun-raj-kuppala) [reply var/spack/repos/builtin/packages/rocm-gdb/package.py:31]: Yes, its a hard dependency.

**Comment by arjun-raj-kuppala** [comment]
@adamjstewart , Can this be merged ? Let me know if anything is pending...

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**